src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.SignVerify.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+using Microsoft.Win32.SafeHandles;
+
+using Internal.Cryptography;
+
+using ErrorCode = Interop.NCrypt.ErrorCode;
+using AsymmetricPaddingMode = Interop.NCrypt.AsymmetricPaddingMode;
+
+namespace System.Security.Cryptography
+{
+    public sealed partial class ECDsaCng : ECDsa
+    {
+        /// <summary>
+        ///     Computes the signature of a hash that was produced by the hash algorithm specified by "hashAlgorithm."
+        /// </summary>
+        public override byte[] SignHash(byte[] hash)
+        {
+            if (hash == null)
+                throw new ArgumentNullException("hash");
+
+            int estimatedSize;
+            switch (KeySize)
+            {
+                case 256: estimatedSize = 64; break;
+                case 384: estimatedSize = 96; break;
+                case 521: estimatedSize = 132; break;
+                default:
+                    // If we got here, the range of legal key sizes for ECDsaCng was expanded and someone didn't update this switch.
+                    // Since it isn't a fatal error to miscalculate the estimatedSize, don't throw an exception. Just truck along.
+                    estimatedSize = 132;
```

Comment:
The first two are exactly KeySize / 4, the third one comes in a little shallow on that approach.  But maybe KeySize / 4 is a better default than 132?